File: internal/model/uniswapcallback/three_sender.go
package uniswapcallback

// ThreeSender ThreeSender实体
type ThreeSender struct {
	ChainID     int16  `gorm:"column:chain_id;not null" json:"chainId"`
	Address     string `gorm:"primary_key;column:address;type:varchar(66)" json:"address"`
	CodeGot     int16  `gorm:"column:code_got" json:"codeGot"`
	HasCallback int16  `gorm:"column:has_callback" json:"hasCallback"`
	Status      int    `gorm:"column:status" json:"status"`
}

// TableName 设置表名
func (ThreeSender) TableName() string {
	return "three_sender"
}

File: src/main/java/beholder_daemon/entity/uniswapCallback/ThreeSender.java
package beholder_daemon.entity.uniswapCallback;


import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Data
@Table(name = "three_sender")
public class ThreeSender {
    @Column(name = "chain_id", nullable = false)
    private Short chainId;
    
    @Id
    @Column(name = "address", length = 66)
    private String address;
    
    @Column(name = "code_got")
    private Short codeGot;
    
    @Column(name = "has_callback")
    private Short hasCallback;
    
    @Column(name = "status")
    private Integer status;

}
